### ISSUE TITLE
Export button on member directory

### ISSUE THREAD

**Issue by canheier** [givingforum/givingforum#204]
SCG worked with themers to create an export button for the member directory. It works on organizations, contacts, and search results. You can check it out here: https://www.socalgrantmakers.org/directory. Please note if you are interested in adding this to your RA's site.

**Comment by AClay2** [comment]
If logged in, is more information included in the Excel spreadsheet other than just the name of the foundation and their website?

[PhilanthropySouthwest]

adrienne Clay
Office Manager & Database Administrator
Phone 214-740-1787
Fax 214-740-1790
[facebook]<https://www.facebook.com/PhilanthropySouthwest>[linkedin]<https://www.linkedin.com/company/458770>[twitter]<https://twitter.com/@southwestfdns>

**Comment by vrozansky** [comment]
Yes. The export changes depending on the user role and what that user can see. So active members get all the fields that they see in the directory.

SCG went even further and added this functionality to individual contacts and organizations.

We like this feature at the Forum and would join the group of RAs interested in this. The total cost is about $1,500 and it'll be an additional hour or so to add it to each individual site. I'm sure you can do the math on your own, but if all RAs are interested in this feature, it'll be about about $75 each. Please vote. Thank you.

**Comment by alientezar** [comment]
Forefront is interested.


Ali Entezar
Director, Information Technology

Forefront
208 South LaSalle, Suite 1540
Chicago, IL 60604
312-327-8932
aentezar@myforefront.org<mailto:aentezar@myforefront.org>
[cid:image001.png@01D15452.F3DAF6C0]<http://myforefront.org/>

Donors Forum is now Forefront.

**Comment by AClay2** [comment]
PSW is interested.

[PhilanthropySouthwest]

adrienne Clay
Office Manager & Database Administrator
Phone 214-740-1787
Fax 214-740-1790
[facebook]<https://www.facebook.com/PhilanthropySouthwest>[linkedin]<https://www.linkedin.com/company/458770>[twitter]<https://twitter.com/@southwestfdns>

**Comment by PhilOhio** [comment]
POH is interested.

**Comment by mtrimbach** [comment]
CNJG is not interested.

**Comment by MainePhilanthropyCenter** [comment]
MPC is not interested right now, but maybe interested in the future if an active member would be able to export search results.

**Comment by kristenruff** [comment]
PNY not interested. Very cool functionality though!

**Comment by sgeis** [comment]
IPA is not interested at this time.

**Comment by kcifone** [comment]
CCP is not interested at this time but may be in the future.

**Comment by EmilyBaron** [comment]
GWP is not interested at this time.

**Comment by RebekahSeder** [comment]
WRAG is not interested.
